resources/maintenance.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html>
+<head>
+<title>Site Maintenance - Study Association Thalia</title>
+<style>
+  body { text-align: center; padding: 150px 20px 0 20px; }
+  h1 { font-size: 50px; }
+  svg { height: 200px; }
+  body { font: 20px Helvetica, sans-serif; color: #333; }
+  article { display: block; text-align: left; max-width: 650px; margin: 0 auto; }
+  a { color: #E62272; text-decoration: none; }
+  a:hover { color: #333; text-decoration: none; }
+	.st0{fill:#E62272;}
+</style>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+<article>
+<svg version="1.1" id="image" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="-233 408.5 374 375" style="enable-background:new -233 408.5 374 375;" xml:space="preserve">
+<g>
+	<path id="hoekje_onder_3_" class="st0" d="M25.3,712.7c-4.4,0,89.8,10.9,91.2,10.9l-51.6,40.1L25.3,712.7L25.3,712.7z"/>
```

Comment:
Panic!!! We have introduced more Dutch language on the concrexit site 😱😱😱🚨🚨🚨

Reply:
Lol